recipes/libtiff/all/conanfile.py

```diff
@@ -104,8 +116,12 @@ def _configure_cmake(self):
             self._cmake.definitions["jpeg"] = self.options.jpeg != False
             self._cmake.definitions["jbig"] = self.options.jbig
             self._cmake.definitions["zlib"] = self.options.zlib
-            self._cmake.definitions["zstd"] = self.options.get_safe("zstd", False)
-            self._cmake.definitions["webp"] = self.options.get_safe("webp", False)
+            if tools.Version(self.version) >= "4.2.0":
+                self._cmake.definitions["libdeflate"] = False # TODO: add libdeflate support (since 4.2.0)
```

Comment:
should we add the option (default false) and throw when it's true so new package id are generated right away?

Reply:
eventually yes

Reply:
done